### PR TITLE
#1401 user entity framework stores

### PR DESCRIPTION
**Pull request by rserj**
There is one module `OrchardCore.Users.EntityFrameworkCore` and Demo feature
`OrchardCore.Demo.Users.EntityFrameworkCore`
You can enable Demo feature and start installation all Users, Roles and Permissions will be stored in tables which will be created by Migration

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Demo/Module.txt [hunk 1]
```diff
@@ -6,3 +6,9 @@ Version: 2.0.x
 OrchardVersion: 2.0.x
 Description: Test
 Category: Samples
+Features:
+    OrchardCore.Demo.Users.EntityFrameworkCore:
+        Name: Users Demo (Entity Framework Store)
+        Description: Use Entity Framework Core ORM in order to store User data
+        Category: Samples
+        Dependencies: OrchardCore.Users.EntityFrameworkCore
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Demo/Module.txt]: For infos

A module can have only one default feature but as soon that you have a `Features:` section, this default feature definition is ignored. So, here we need to define it as a 1st feature in this section.

Note: When not defined in a feature, `Priority, Category, Description` are grabbed from the module.

So, here we would need, e.g.

    Name: Orchard Demo
    AntiForgery: enabled
    Author: The Orchard Team
    Website: http://orchardproject.net
    Version: 2.0.x
    OrchardVersion: 2.0.x
    Features:
        OrchardCore.Demo:
            Name: Orchard Demo
            Description: Test
            Category: Samples
        OrchardCore.Demo.Users.EntityFrameworkCore:
            Name: Users Demo (Entity Framework Store)
            Description: Use Entity Framework Core ORM in order to store User data
            Category: Samples
            Dependencies: OrchardCore.Users.EntityFrameworkCore

File: src/OrchardCore.Modules/OrchardCore.Users.EntityFrameworkCore/Module.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name: Users (Entity Framework Core stores)
+AntiForgery: enabled
+Author: The Orchard Team
+Website: http://orchardproject.net
+Version: 2.0.x
+OrchardVersion: 2.0.x
+Description: The Users Entity Framework Core module allows to store user data via Entity Framework Core ORM
+FeatureDescription: Entity Framework Core Stores for the Users module.
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.Users.EntityFrameworkCore/Module.txt]: For infos.

For the module default feature or for a feature defined in the features section, the description key is `Description`. Currently, through the code the `FeatureDescription` key is never used. E.g, in the `Modules` page only `Description` values of features (could be the module default one) are showed up.

So, i'm used to remove the `FeatureDescription` line and just use the `Description` line with the most descriptive text.

File: src/OrchardCore.Modules/OrchardCore.Demo/Module.txt (hunk 1 above)
Comment (rserj) [reply src/OrchardCore.Modules/OrchardCore.Demo/Module.txt]: Thank you for review and explanation, I'm fixing it right now

**Comment by sebastienros** [comment]
Do we need to resurrect this PR? Do you need to rewrite it?

**Comment by agriffard** [comment]
OrchardCore.Users.EntityFrameworkCore does not exist anymore.

**Pull request closed without merging**